packages/graphql-server/src/index.ts

```diff
@@ -23,7 +23,7 @@ import { parseBody } from './parse-body'
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export type RootResolver<E extends Env = any, P extends string = any, I extends Input = {}> = (
   // eslint-enable-next-line @typescript-eslint/no-explicit-any
-  c?: Context<E, P, I>
+  ctx: Context<E, P, I>
```

Comment:
IMO: In PR #640, I changed the name to align with the variable naming conventions used in other projects within the Hono organization. Therefore, I believe it is more appropriate to stick with `c` rather than changing it to `ctx`.